webui/react/src/pages/WorkspaceList.tsx

```diff
@@ -394,15 +394,15 @@ const WorkspaceList: React.FC = () => {
           workspacesList
         ) : settings.whose === WhoseWorkspaces.All && settings.archived && !isLoading ? (
           <Message
-            data-testid="noWorkspaces"
             description={`Create a workspace to keep track of related projects and ${f_flat_runs ? 'searches' : 'experiments'}.`}
             icon="workspaces"
+            testId="noWorkspaces"
             title="No Workspaces"
           />
         ) : (
           <Message
-            data-testid="noMatchingWorkspaces"
             icon="warning"
+            testId="noMatchingWorkspaces"
```

Comment:
did the models get updated too?

Reply:
don't think they needed to be updated, they were already looking for this testId, but the hew component was not rendering with it.